client/src/config/constants.js

```diff
@@ -1 +1 @@
-export const ENDPOINT = "http://digitalbraininjury.com/api/" ;
+export const ENDPOINT = "http://www.digitalbraininjury.com/api/" ;
```

Comment:
can the domain made into a variable and passed in from configuration file? Soon we will be moving to nsfcareer.io